packages/twenty-server/src/database/typeorm/core/migrations/billing/1732710668000-billingTableMigrationTransfer.ts

```diff
@@ -0,0 +1,100 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class BillingTableMigrationTransfer1732710668000
+  implements MigrationInterface
+{
+  name = 'BillingTableMigrationTransfer1732710668000';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    try {
+      await queryRunner.query(
+        `ALTER TABLE "core"."billingSubscriptionItem" ALTER COLUMN "deletedAt" TYPE TIMESTAMP WITH TIME ZONE USING "deletedAt" AT TIME ZONE 'UTC'`,
+      );
+      await queryRunner.query(
+        `ALTER TABLE "core"."billingSubscription" ALTER COLUMN "deletedAt" TYPE TIMESTAMP WITH TIME ZONE USING "deletedAt" AT TIME ZONE 'UTC'`,
+      );
+    } catch (error) {
+      // Column may have already been altered in UseTimestampWithTZ1711633823798
+      // so we don't need to do anything
+    }
+
+    // From UpdateInconsistentUserConstraint1715593226719
+
+    try {
+      await queryRunner.query(
+        `ALTER TABLE "core"."billingSubscription" ALTER COLUMN "status" TYPE text`,
+      );
+      await queryRunner.query(
+        `ALTER TABLE "core"."billingSubscription" ALTER COLUMN "interval" TYPE text`,
+      );
+    } catch (error) {
+      // Column may have already been altered in UseEnumForSubscriptionStatusInterval1719327438923
```

Comment:
I don't get these comments ?

Reply:
got it

Reply:
I don't see a better way (we could check table existence before but this looks a bit painful to do)